src/Modules/ContentBlocks/Domain/ContentBlock/Command/ChangeContentBlockHandler.php

```diff
@@ -5,41 +5,29 @@
 use ForkCMS\Core\Domain\MessageHandler\CommandHandlerInterface;
 use ForkCMS\Modules\ContentBlocks\Domain\ContentBlock\ContentBlock;
 use ForkCMS\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockRepository;
+use ForkCMS\Modules\ContentBlocks\Domain\ContentBlock\Service\UpdateContentBlockWidget;
 use ForkCMS\Modules\Frontend\Domain\Block\Block;
 use ForkCMS\Modules\Frontend\Domain\Block\BlockRepository;
 
 final class ChangeContentBlockHandler implements CommandHandlerInterface
 {
     public function __construct(
         private readonly ContentBlockRepository $contentBlockRepository,
-        private readonly BlockRepository $blockRepository
+        private readonly BlockRepository $blockRepository,
+        private readonly UpdateContentBlockWidget $widget
```

Comment:
Not a fan of this, just implement it with the repository in this command, this is not something that is reused somewhere else